docs/self-managed/operational-guides/configure-flow-control/configure-flow-control.md

```diff
@@ -2,29 +2,27 @@
 id: configure-flow-control
 title: "Configure flow control"
 sidebar_label: "Configure flow control"
-description: "Learn how to configure flow control"
+description: "Configure flow control to prevent building an excessive exporting backlog."
 ---
 
-When a broker receives a client request, it is first written to the **event stream** (see [internal processing](/components/zeebe/technical-concepts/internal-processing.md) for details), and later processed by the stream processor.
+When the processing speed of new requests is faster than the rate at which they are exported, backlogs of unexported records can occur. Flow control slows the write rate of new records, and prevents the stream from building an excessive backlog of records not yet exported. By default, flow control attempts to achieve 200ms response times.
```

Comment:
I've adjusted this a bit, and include the write rate information - I don't want to get too detailed with internal processing information, as this is meant to be an operational guide, and not too heavy on technical concepts. Some of this could be expanded on as we fold in information from the backpressure guides to make both experiences a bit cleaner, too.